packages/expo-image/CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 
 ### 💡 Others
 
+## 2.0.3 — 2024-11-23
+
+- Update `ImageProps` so `children` are omitted. ([#33afe30](https://github.com/expo/expo/pull/33210) by [@ashaller2017](https://github.com/ashaller2017)) ([#33210](https://github.com/expo/expo/pull/33210) by [@ashaller2017](https://github.com/ashaller2017))
+
+### 💡 Others
+
```

Comment:
Changelog entries should be put under "Unpublished" section. Version and the date is automatically added by the publish script.
```suggestion
- Update `ImageProps` type so `children` are omitted. ([#33210](https://github.com/expo/expo/pull/33210) by [@ashaller2017](https://github.com/ashaller2017))

```